README.md

```diff
@@ -610,37 +612,35 @@ Create a base RHEL data volume prior to deploying OpenStack.  This will be used
 
     The osconfiggenerator created above will automatically generate playbooks any time you scale or modify the ConfigMaps for your OSP deployment. Generating these playbooks takes several minutes. You can monitor the osconfiggenerator's status condition for it to finish.
 
-9) Login to the 'openstackclient' pod and deploy the OSP software via the rendered ansible playbooks. At this point all baremetal and virtualmachine resources have been provisioned within the OCP cluster.
-
-    The `tripleo-deploy.sh` script supports three actions:
-    * `-d` - show the `git diff` of the playbooks to the previous accepted
-    * `-a` - accept the new available rendered playbooks and tag them as `latest`
-    * `-p` - run the ansible driven OpenStack deployment
-
-    a) check for new version of rendered playbooks and accept them
+9) Obtain the latest OsConfigVersion (Ansible Playbooks). Select the hash/digest of the latest osconfigversion for use in the next step.
 
     ```bash
-    oc rsh openstackclient
-    bash
-    cd /home/cloud-admin
+    oc get -n openstack --sort-by {.metadata.creationTimestamp} osconfigversions -o json
+    ```
 
-    # (optional) show the `git diff` of the playbooks to the previous accepted
-    ./tripleo-deploy.sh -d
+   NOTE: OsConfigVersion objects also have a 'git diff' attribute that can be used to easily compare the changes between Ansible playbook versions.
 
-    # accept the new available rendered playbooks (if available) and tag them as `latest`
-    ./tripleo-deploy.sh -a
+10) Create an OsDeploy (executes Ansible playbooks)
+
+    ```yaml
+    apiVersion: osp-director.openstack.org/v1beta1
+    kind: OpenStackDeploy
+    metadata:
+      name: default
+    spec:
+      configVersion: n5fch96h548h75hf4hbdhb8hfdh676h57bh96h5c5h59hf4h88h...
+      configGenerator: default
     ```
-    b) run ansible driven OpenStack deployment
 
-    ```bash
-    oc rsh openstackclient
-    bash
-    cd /home/cloud-admin
+    If you write the above YAML into a file called deploy.yaml you can create the OpenStackDeploy via this command:
 
-    # run ansible driven OpenStack deployment
-    ./tripleo-deploy.sh -p
+    ```bash
+    oc create -f deploy.yaml
     ```
 
+    As the deployment runs it will create a Kubernetes Job to execute the Ansible playbooks. You can
+    tail the logs of this job/pod to watch the Ansible playbooks run.
```

Comment:
should we also describe how to run it manually via the openstackclient for debugging after a deployment might have failed?

Reply:
good call, just pushed a new commit with this in it.